tests/testthat/test-dfm_group.R

```diff
@@ -56,3 +56,94 @@ test_that("dfm.character groups works (#794)", {
         dfm(corp, groups = "grp")
     )
 })
+
```

Comment:
Here I've rewritten (slightly) your tests and added others. These will fail until the function is changed.